src/routes/oath.ts

```diff
@@ -27,19 +28,22 @@ const SAASQUATCH_CLIENT_SECRET = process.env.SAASQUATCH_CLIENT_SECRET;
 
 export const tokenStore: any = {};
```

Comment:
We still need to take out all use of tokenStore but it is used across files and should be done in a follow up ticket.

Reply:
@MAsimSENG will handle this in https://trello.com/c/YWc63K6w/117-refactor-hubspot-oauth